app/models/document.rb

```diff
@@ -0,0 +1,11 @@
+class Document < Mongoid_Base
```

Comment:
Make sure to fix this everywhere else as well -- including the `MongoidBase` model itself.